src/components/nodes/pages/sup-book/digital-project-page/digital-project-page.tsx

```diff
@@ -221,7 +221,7 @@ const DigitalProjectPage = async ({node, ...props}: Props) => {
           </div>
         </div>
 
-        <div className="md:w-1/4] md:order-first">
+        <div className="md:order-first md:w-1/4">
```

Comment:
on a mobile device, and we make the image bigger? 
![image](https://github.com/user-attachments/assets/9126006c-11fd-47e6-85ff-f78bd073c6e8)